src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ServiceCategoryFullDTO.kt

```diff
@@ -21,15 +21,15 @@ data class ComplexityLevelDTO(
   val description: String,
 )
 
-data class ServiceCategoryDTO(
+data class ServiceCategoryFullDTO(
```

Comment:
Could we rename 1) ServiceCategoryDTO as ServiceCategoryBaseDTO and 2) ServiceCategoryFullDTO as ServiceCategoryDTO, and also, inherit one from the other - but understand if it causes too much change